src/app/interfaces/strongRoute.ts

```diff
@@ -192,18 +221,37 @@ export class StrongRoute {
    * Example output: "/home/house"
    */
   public toString(): string {
-    return "/" + (this.fullRoute ? this.fullRoute : "");
+    // Prevent double '/' at start of route string
+    const path = this.toRoute().join("/");
+    return path.length >= 2 ? path.substr(1) : path;
```

Comment:
maybe better expressed with 
```suggestion
    return path.startsWith('/')  ? path.substr(1) : path;
```